src/docs/product/alerts-notifications/metric-alerts.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Metric Alerts
+sidebar_order: 1
+redirect_from:
+  - /workflow/alerts-notifications/alerts/
+  - /workflow/notifications/alerts/
+  - /product/alerts-notifications/alerts/
+---
+
+MIMI NOTE: this is true for all alerts. Maybe put it in the index page: Sentry users with admin permissions or higher can create alerts. To confirm or set admin permissions, navigate to **Project Settings > Alerts > New Alert**. To change the frequency of your alerts, configure your [Alert Subscription](/product/alerts-notifications/alert-subscription/).
+
+## Metric Alerts
+
+MIMI NOTE: check the grammar below:
+
+Metric alerts are typically used when you need to know about events such as a spike in the overall number of errors in a project, or a change in a performance metric like [latency](/product/performance/metrics/#latency), [Apdex]((/product/performance/metrics/#apdex)), [failure rate](/product/performance/metrics/#failure-rate), and [throughput](/product/performance/metrics/#throughput-total-tpm-tps).
+
+### Alert Builder
+
+The alert builder sets the metric alert's triggers: Critical (required) and Warning (optional). The warning trigger's threshold must be breached before the critical trigger will set in motion. Triggers are evaluated approximately every minute from the highest severity to lowest.
+
+Sentry creates an alert with the severity of the matched trigger (warning or critical). If an alert is already active, its status is updated. If no resolution threshold is specified, the alert will automatically resolve when it's no longer violating the critical or warning conditions. You can also resolve alerts manually by navigating to **Project Settings > Alerts**.
+
+An alert occurs when it is initially triggered or if it changes status. Afterward, the alert will perform a configured action, such as:
```

Comment:
I think the phrase "is created" is a bit unclear. The user doesn't "create" the alert. Sentry "creates" the alert when a rule is triggered. I also wanted to make it clear that the alert actions are configurable. Currently, it's not clear _how_ the actions are associated with the trigger. They're associated with the trigger because they're configured by the user.